tests/e2e/scenarios/test_reborn_webui_v2_legacy_tool_permissions.py

```
@@ -193,6 +193,19 @@ def _tool_row(page, name: str):
     return page.locator(SEL_V2["settings_tool_row_for"].format(name=name))
 
 
+def _permission_button(page, name: str):
+    return _tool_row(page, name).locator(SEL_V2["settings_tool_permission"])
+
+
+async def _choose_permission(page, name: str, label: str):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_choose_permission`

(ANN202)